registry/components/magicui/gauge-circle.tsx

```diff
@@ -0,0 +1,130 @@
+import { cn } from "@/lib/utils";
+
+interface Props {
+  max: number;
+  value: number;
+  min: number;
+  gaugePrimaryColor: string;
+  gaugeSecondaryColor: string;
+  className?: string;
+}
+
+/*
+  This is the css that should be added to a css file
+ */
+const css = `.gauge_circle {
+--transition-length: 1s;
+--transition-step: 200ms;
+--delay: 0s;
+--percent-to-deg: 3.6deg;
+transform: translateZ(0);
+} 
+
+.gauge_animate .gauge_arc {
+    stroke-dasharray: calc(var(--stroke-percent) * var(--percent-to-px)) var(--circumference);
+    transition: var(--transition-length) ease var(--delay),stroke var(--transition-length) ease var(--delay);
+    transition-property: stroke-dasharray,transform;
+    transform: rotate(calc(-90deg + var(--gap-percent) * var(--offset-factor) * var(--percent-to-deg)));
+    transform-origin: calc(var(--circle-size) / 2) calc(var(--circle-size) / 2)
+}
+
+.gauge_animate .gauge_arcSecondary {
+    --offset-factor-secondary: calc(1 - var(--offset-factor));
+    stroke-dasharray: calc(var(--stroke-percent) * var(--percent-to-px)) var(--circumference);
+    transform: rotate(calc(1turn - 90deg - (var(--gap-percent) * var(--percent-to-deg) * var(--offset-factor-secondary)))) scaleY(-1);
+    transition: all var(--transition-length) ease var(--delay);
+    transform-origin: calc(var(--circle-size) / 2) calc(var(--circle-size) / 2)
+}
+
+.gauge_animate .gauge_content {
+    animation: gauge_fadeIn var(--transition-length) ease forwards;
+    animation-delay: var(--delay)
+}
+
+@keyframes gauge_fadeIn {
+    to {
+        opacity: 1
```

Comment:
this should be added to a tailwind.config.js file and then also written in the `gauge-circle.mdx` so users know how to set it up

Reply:
Got it. I'll add the  to the  file and update the  documentation so users know how to set it up.

Reply:
No need to add the @keyframes gauge_fadeIn in the tailwind.config.js file, I solved it this way.
![image](https://github.com/magicuidesign/magicui/assets/101147375/2e04f7a7-4d6c-46b7-9bc2-910199d1d1de)